tests/node_tests/backbeat/tests/crr/awsBackend.js

```diff
@@ -23,6 +23,7 @@ const REPLICATION_TIMEOUT = 300000;
 
 describe('Replication with AWS backend', function() {
     this.timeout(REPLICATION_TIMEOUT);
+    this.retries(3);
```

Comment:
I've noticed HTTP 503 status code (Service Unavailable) when Cloudserver tries to put objects to AWS during CRR. It seems like a retry of the test is warranted for that case rather than a CI failure, though I'd ideally like to minimize reliance on retries.